source/applications/dependency-injection.md

```diff
@@ -209,3 +215,28 @@ export default {
   initialize: initialize
 };
 ```
+
+### Getting an Application Instance from a Factory Instance
+
+[`Ember.getOwner`][4] will retrieve the application instance from an Ember
+Object that was instantiated from a registered factory. This means that
+framework objects like components, helpers, and routes can use
+[`Ember.getOwner`][4] to perform lookups through their application instance.
+
+```app/helpers/logger.js
+import Ember from 'ember';
+
+const { getOwner } = Ember;
+
+export default Ember.Helper.extend({
+  init() {
+    this._super(...arguments);
+
+    this.errorReporter = getOwner(this).lookup('error-reporter:sentry');
```

Comment:
Yeah this logging example wasn't working. I stole the audio example from the Ember 2.3 beta release blog instead and made a few minor changes. If you have any other nice use cases in mind let me know.

My current use case for `getOwner` is to dig into the registry and pull out private Ember internals -- not an exemplary use case.